openwisp_utils/admin.py

```diff
@@ -100,6 +103,77 @@ class Media:
     uuid.short_description = _('UUID')
 
 
+class CopyableFieldError(FieldError):
+    pass
+
+
+class CopyableFieldsAdmin(ModelAdmin):
+    """
+    An admin class that allows us to set read-only input fields
+    (to make it easy and quick to copy/paste).
+    """
+
+    copyable_fields = ()
+    change_form_template = 'admin/change_form.html'
+
+    def uuid(self, obj):
+        return obj.pk
```

Comment:
Yes, we can inherit the `UUIDAdmin` class for the `copyable` feature, but we must remove `uuid.js` from the `UUIDAdmin` otherwise, it will modify the same input field `uuid` that is later handled by `CopyableAdmin`,resulting in an empty input value for the `uuid` field, as shown below: 

![Screenshot from 2023-03-01 15-45-37](https://user-images.githubusercontent.com/56113566/222115664-1cefe827-49bf-4a08-b98e-a32ed253c6c2.png)

After removing the media class of `UUIDAdmin`, we get the expected results, see below:

```py

class UUIDAdmin(ModelAdmin):  # pragma: no cover
         fields = super().get_readonly_fields(request, obj)
         return self._process_fields(fields, request, obj)
	
	# after removing media class
-    class Media:
-    		js = ('admin/js/jquery.init.js', 'openwisp-utils/js/uuid.js')
```

![Screenshot from 2023-03-01 16-09-59](https://user-images.githubusercontent.com/56113566/222116100-b8525e79-6031-446e-89f8-ee1aa0341572.png)